cmd/cmd.go

```diff
@@ -50,6 +51,10 @@ func pull(model string) error {
 		context.Background(),
 		&api.PullRequest{Model: model},
 		func(progress api.PullProgress) error {
+			if progress.Error.Code == http.StatusBadGateway {
+				// couldn't pull the model from the directory, proceed in offline mode
+				return nil
+			}
```

Comment:
IMO this should exit on any error and fall back to call generate. That way, a user can create a file in `~/.ollama/models` and run it by name without specifying a full path